src/styles/blocks/header.scss

```diff
@@ -0,0 +1,61 @@
+.header {
+  background-image: url("./src/images/photos/header-mobile.png");
+  background-size: cover;
+  background-position: right;
+
+  @include on-tablet {
+    background-image: url("./src/images/photos/header-tablet.png");
+  }
+
+  @include on-desktop {
+    background-image: url("./src/images/photos/header.png");
+  }
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    height: 100vh;
+
+    grid-column: 1 / -1;
+  }
+
+  &__title {
```

Comment:
I'd add a `cursor: default` property, it just looks better, especially for header.